src/NUnitFramework/tests/Api/FrameworkControllerTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// ***********************************************************************
+// ***********************************************************************
```

Comment:
EOL change I think?
Sorry, there's a few in this PR from files I've opened. Please let me know if I should revert.

Reply:
This is fine. It likely is removing the byte order marks. This is because people have saved all the files with random character encodings over the years and we recently added `charset = utf-8` to our .editorconfig to bring us toward consistency.

Reply:
Ah that makes sense. Ok cool, thanks for clarifying.